### PR TITLE
Aggregate expert Test and docs fixes

### PR DESCRIPTION
**Pull request by wagner-certat**
Please especially check if `counter_ignored` is implemented correctly

### CODE REVIEW

File: intelmq/bots/experts/aggregate/expert.py [hunk 1]
```diff
@@ -46,8 +51,14 @@ def cleanup(self):
                     event.add("extra.count", int(data['c']))
                     event.add("extra.time_end", data['l'])
                     self.send_message(event)
+                    counter_sent += 1
+                else:
+                    counter_ignored += 1
```
Comment (wagner-certat) [inline intelmq/bots/experts/aggregate/expert.py:56]: Maybe this case here is "message gets discarde because threshold was not reached"?

Comment (waldbauer-certat) [reply intelmq/bots/experts/aggregate/expert.py:56]: Yes, if threshold is not reached, message gets dropped.

**Review by waldbauer-certat — APPROVED** [review]

File: intelmq/bots/experts/aggregate/expert.py [hunk 2]
```diff
@@ -46,8 +51,14 @@ def cleanup(self):
...
                 self.cache_get_redis_instance().delete(key)
             self.__next_cleanup = int(time.time()) + 10
+            self.logger.debug('Completed Cleanup. Messages sent: %d, messages ignored: %d.', counter_sent, counter_ignored)
+        else:
+            self.logger.debug('Skipped Cleanup (%fs < 10s).', self.__next_cleanup - time.time())
```
Comment (waldbauer-certat) [inline intelmq/bots/experts/aggregate/expert.py:61]: This might get spammy in the logs, because each event received will run the cleanup process.

Comment (wagner-certat) [reply intelmq/bots/experts/aggregate/expert.py:61]: OTOH it's the debug log, which logs *at least* one line for every incoming message anyway

**Review by waldbauer-certat — APPROVED** [review]

**Pull request merged**